src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,39 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const selectedIdCheck = (
+    tabs.find(tab => tab.id === selectedTabId) !== undefined
+      ? selectedTabId
+      : tabs[0].id
+  );
```

Comment:
```suggestion
  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
```
This part can be simplified